pkg/operator/resources/validations.go

```diff
@@ -147,21 +147,23 @@ func ValidateClusterAPIs(apis []userconfig.API, projectFiles spec.ProjectFiles)
 CPU Reservations:
 
 FluentBit 100
-StatsD 100
+StatsDExporter 100
+NodeExporter 100
 KubeProxy 100
 AWS cni 10
 Reserved (150 + 150) see eks.yaml for details
 */
-var _cortexCPUReserve = kresource.MustParse("610m")
+var _cortexCPUReserve = kresource.MustParse("710m")
```

Comment:
We should confirm that it's still possible to run on `t3.small` (otherwise we can black list it like we do with micro and nano)